docs/tutorials/state_histograms.ipynb

```diff
@@ -0,0 +1,293 @@
+{
```

Comment:
<p><code>used process</code>  -&gt; <code>used to process</code></p><p> <code>genrate</code>  -&gt; <code>generate</code></p><p><code>of parity of number</code> -&gt; <code>of the parity of the number</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4024/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
done